src/sharedComponents/Footer.jsx

```diff
@@ -1,27 +1,44 @@
 import React from "react";
 import Logo from "../assets/logoes/logo-negative.png";
+import UnconLogo from "../assets/logoes/uncon-negative.png";
+import CifarLogo from "../assets/logoes/cifar.png";
 import { Text } from "./Reusble";
 import { ReactComponent as Drorsoft } from "../assets/logoes/drorsoft-logo.svg";
 import classNames from "classnames";
 import { footerHeight } from "../Utils/HardCoded";
+import { Site } from "../config/siteType";
 
 export default function Footer({ isFixed }) {
+  const isUncontrast = Site.type === "uncontrast";
   return (
     <div
       className={classNames(
         `${
           isFixed && "absolute bottom-0"
-        }h-max-${footerHeight} w-full flex flex-col sm:flex-row justify-between items-center py-4 sm:px-16 bg-black rounded-t-lg sm:rounded-none`
+        } h-[${footerHeight}px] w-full flex flex-col sm:flex-row justify-between items-center py-4 sm:px-16 bg-black rounded-t-lg sm:rounded-none`
```

Comment:
You should make sure this can work  because tailwind builds all the css **before** the JS runs. So it might not know which class to build